content/pop-basics.md

```diff
@@ -17,30 +17,169 @@ section: pop
 tableOfContents: true
 ---
 
+Welcome to the Pop!_OS desktop! By default, it's clean and ready for action.
 The Pop!_OS operating system provides all the features you would expect from a modern desktop.
 
-## Getting to Know The Pop!_OS Desktop
+Pop!\_OS is based on Ubuntu, however, extra features and tools have been added, while other aspects have been slimmed down. You can read more about Pop!\_OS features and the differences between Pop!\_OS and Ubuntu [here](/articles/difference-between-pop-ubuntu)
 
-![Pop Desktop](/images/pop-basics/pop-desktop.png)
+## Pop!_OS Desktop Environment
 
-Welcome to the Pop!_OS desktop! By default, it's clean and ready for action.
+The Pop!_OS Desktop environment is based on GNOME shell and layered with other components called **Pop!\_Shell** and **C.O.S.M.I.C. (Computer Operating System Main Interface Components)** For more info, including source-code, refer to [this page](https://github.com/pop-os/).
```

Comment:
Isn't Source Code two words?

Reply:
I think it could go either way, but I'll do "source code" instead.